packages/components/src/internal/constants.ts

```diff
@@ -20,6 +20,7 @@ import { getServerContext } from '@labkey/api';
 // but hopefully these can become their own packages or part of this package directly
 export const LABKEY_VIS = getServerContext().vis;
 export const LABKEY_WEBSOCKET = getServerContext().WebSocket;
+export const LABKEY_LOGIN = getServerContext().login;
```

Comment:
As mentioned in the `@labkey/api` PR this would become `getServerContext().PasswordGauge`. That said, perhaps we just reference it directly where it is used rather than declaring this global constant?

Reply:
I left this in so that these consts are all next to eachother to give us an idea of which of these types of namespaces we are using off of getServerContext().